### PR TITLE
Adding German Language Translations 

### PR DESCRIPTION
**Pull request by Sonichigo**
# Description
Adding Translations of README and CONTRIBUTING file. And Updating the Links for the Same.

# Context
German is a West Germanic language that is mainly spoken in Central Europe. It is the most widely spoken and official or co-official language in Germany

Signed-off-by: Animesh Pathak <pathakanimesh12@gmail.com>

### CODE REVIEW

**Comment by Sonichigo** [comment]
Signed-off-by: Animesh Pathak <kurosakiichigo.songoku@gmail.com>

**Comment by Sonichigo** [comment]
Signed-off-by: Animesh Pathak pathakanimesh12@gmail.com

File: translations/README.md [hunk 1]
```diff
@@ -10,4 +10,5 @@
 | 🇨🇳 | [中文](README.zh.md) |
 | 🇫🇷 | [Français](README.fr.md) |
 | :india: | [ગુજરાતી](README.gu.md) |
+| GU | [Gujarati](README.gu.md)
 | GE | [German](README.ge.md)
```
Comment (Sonichigo) [inline translations/README.md:14]: Signed-off-by: Animesh Pathak pathakanimesh12@gmail.com

**Comment by Sonichigo** [comment]
@akhilerm please review!

**Comment by kmova** [comment]
@Sonichigo 

The translations/README.md is a placeholder for listing translations. this will avoid enumerating all languages in the main page and every time modifying all the translated files. That file should not be deleted. 


Can you modify this PR by reverting the changes made to README.md and CONTRIBUTING.md and also translate the latest README.md, with the proper links to the translations/README.md.

**Comment by kmova** [comment]
@Sonichigo - would you like to raise a new PR with only the german translation files, without having conflicts with others. Ideally, you would just have only one file to commit.

**Comment by kmova** [comment]
@Sonichigo - there is already a german translation file. https://github.com/openebs/openebs/blob/master/translations/README.de.md

**Pull request closed without merging**